### ISSUE TITLE
rotors_gazebo_plugins fails to build

### ISSUE THREAD

**Issue by HusseinHazem** [ethz-asl/rotors_simulator#477]
![screenshot from 2018-04-05 20 12 59](https://user-images.githubusercontent.com/36478570/38383888-fbf629e0-390d-11e8-813b-efb83f0a895b.png)

**Comment by marco-tranzatto** [comment]
Did you follow [these instructions](https://github.com/ethz-asl/rotors_simulator/wiki/Setting-up-the-RotorS-Simulator) (point number 3)?
It looks like you didn't install Rotors dependencies (i.e. `glog_catkin`).

**Comment by HusseinHazem** [comment]
Yes i installed all the packages required from the mpc library which are the same as the one you mentioned

**Comment by aeudes** [comment]
Not sure if it the same problem but we recently have problem with the ability to find gtest on new install.
It was related to "recent" change on catkin.
You could try to apply this commit: https://github.com/ros/catkin/pull/927/files   on /opt/ros/kinetic/share/catkin/cmake/test/gtest.cmake

**Comment by HusseinHazem** [comment]
i am not sure what you are saying tbh aeudes, and i installed the package glog_catkin in the src and even more packages won't build and it itself failed to build

**Comment by ffurrer** [comment]
So you did `catkin build glog_catkin` and that succeded or not?
If you did only after installing other packages you probably want to clean your workspace and try re-building everything.
```
catkin clean
catkin build
```

**Comment by HusseinHazem** [comment]
I did exactly that

**Comment by m-erdem** [comment]
Which gazebo version do you use? As much as I know the package is not compatible with gazebo 9 and 8. I had the same error with version 9, but not quite sure with gazebo8. You may wanna use gazebo7.

**Comment by HusseinHazem** [comment]
gazebo 7

**Comment by HusseinHazem** [comment]
![screenshot from 2018-04-27 14 01 24](https://user-images.githubusercontent.com/36478570/39362535-18e4f57a-4a27-11e8-88f7-0f7c1a4ce7c6.png)

**Comment by helenol** [comment]
Make sure you have mavros in your workspace:
https://github.com/ethz-asl/mavros
And that it's built. If you still get this error, do ```catkin clean --all``` and rebuild your entire workspace.

**Comment by CarSanoja** [comment]
Hello!
I am getting exactly the same error. Somebody solved?

**Comment by anushmanukyan** [comment]
Hello, I have a similar issue. I have done all possible steps, but I can not manage to build rotors_gazebo_plugins package. 
I have ubuntu 16.04, gazebo7.
At the end I can launch an example like mav.launch, but then there is no ROS messages that the mav provides, i can see only messages of gazebo.

Anybody could give a hint what could be the issue? 

Thanks in advance.

**Comment by wuyuzaizai** [comment]
i got the same error

**Comment by anushmanukyan** [comment]
I used docker, and all worked perfectly.
